django/contrib/admin/sites.py

```diff
@@ -154,6 +156,12 @@ def actions(self):
         """
         return six.iteritems(self._actions)
 
+    def get_empty_value_display(self):
+        return self.empty_value_display
+
+    def set_empty_value_display(self, empty_value_display):
```

Comment:
I dislike the Java like getter and setter style. Can't you turn it into a `@property` and store `'-'` in `_empty_value_display`?